client/src/components/app/App.js

```diff
@@ -1,34 +1,61 @@
 import "./App.css";
 import { Switch, Route, NavLink } from "react-router-dom";
+import axios from "axios";
+import { useState, useEffect } from "react";
 import AppHeader from "./AppHeader.js";
-import HomePage from "../../pages/HomePage.js";
-// import AboutPage from "../../pages/AboutPage.js";
-import FavsPage from "../../pages/FavsPage.js";
-import BooksPage from "../../pages/BooksPage.js";
-import BookPage from "../../pages/BookPage.js";
+
+import MyCommunitiesPage from "../../pages/MyCommunitiesPage.js";
+
+import CommunitiesContext from "../../contexts/CommunitiesContext";
+
+import Modal from "../modal/Modal.js";
+import NewCommunityModalPage from "../../pages/NewCommunityModalPage.js";
 
 function App() {
+  const [resources, setResources] = useState([]);
+
+  const primaryAddActionFormId = "newCommunity";
+
+  useEffect(() => {
+    console.log("yohooo effect");
```

Comment:
No console logs in PR :P

Reply:
A modal dialog is sort of a "pop up" that opens in a separate layer above the current page and remains open until the user closes it explicitly -  i.e. it prevents interaction with any other UI (i.e. controls of the underlying page) until it is dismissed. I use modals here primarily to give the user the impression, that he doesn't need to leave the current page and can do stuff on the fly  while all new items or edits to existing items are instantly reflected in the underlying "parent" view ...

Reply:
I'm using a file for css-color-vars - see `styles/_base.css`